lib/manageiq/providers/azure.rb

```diff
@@ -0,0 +1,8 @@
+require "manageiq/providers/azure/engine"
+
+module Manageiq
```

Comment:
Going to merge as is... @durandom Please fix this in a follow up PR and in the generator as well